### ISSUE TITLE
Create the jumbotron

### ISSUE THREAD

**Issue by michaelawetahegn** [siltena/hopr.gov.et#8]
Add support for a jumbotron on the homepage that displays the latest news, live and upcoming events etc. The information that goes into the jumbotron (as well as the design) are determined by what information the users need to see on the site once they open it. To get an idea, you can reference the jumbotron from the https://www.house.gov/ site. It includes:

* Schedules for meetings
* Recent votes
* Bills & Reports

Note to self: the development of the jumbotron will be contained in the `jumbotron-support` branch. Since the `jumbotron-support` branch is the "parent" of the `navbar-support` branch and I will be developing both in parallel, the `navbar-support` branch should regularly be merged into the `navbar-support` branch. Merge into master should involve merging `navbar-support` into master first, then merging `jumbotron-support` second.

**Comment by michaelawetahegn** [comment]
The current design is to split the jumbotron into two and add a carousel on the left side. To build the carousel, use inspirations from the following site: https://mdbootstrap.com/docs/jquery/javascript/carousel/

The best looking option seems to be the one with circle buttons. Additionally, there should be captions that are displayed with a dark background (so that light images don't hide the text).

**Comment by michaelawetahegn** [comment]
Added carousel with placeholder images and data: e631e23de14cd09182499f98c3aace8ac8b0a430 It's missing the dark background behind the caption because, as the 2nd image shows, it's hard to read the text with a light image in the background.

**Comment by michaelawetahegn** [comment]
Finished implementation of the placeholder "jumbotron" here: 70a41c4a199f97b00dee7d1614159d00ba0468a1

The live data, events calendar etc. will need to be added to a separate issue as not to crowd this one.

**Comment by michaelawetahegn** [comment]
Merged 5a16ed210f3319b912ce2686d176e8c6b60acc89
